roofit/roofitcore/test/TestStatistics/testLikelihoodGradientJob.cxx

```diff
@@ -22,15 +22,26 @@
 #include "RooHelpers.h"
 #include "RooMinimizer.h"
 #include "RooFitResult.h"
+#include "RooGenericPdf.h"
 #include "RooFit/TestStatistics/LikelihoodWrapper.h"
 #include "RooFit/TestStatistics/RooUnbinnedL.h"
 #include "RooFit/TestStatistics/RooRealL.h"
 #include "RooFit/TestStatistics/buildLikelihood.h"
 #include "RooFit/MultiProcess/JobManager.h"
 #include "RooFit/MultiProcess/Config.h"
 
+// for MinuitFcnGrad test:
+#include "RooFit/MultiProcess/Config.h"
+#include "RooStats/ModelConfig.h"
```

Comment:
```suggestion
#include "RooFit/ModelConfig.h"
```

The ModelConfig was moved from RooFit to RooStats, because it is used so much also outside the context of RooStats. This should fix the CI build failures

Reply:
Awesome, thanks, fixed and force-pushed.